monitoring/dashboard.py

```diff
@@ -525,6 +542,111 @@ def top10_errors_by_bucket(title, code):
     title="5xx : Top10 by Bucket", code='~"5.."'
 )
 
+
+quotaHealth = GaugePanel(
+    title="Quota service uptime",
+    dataSource="${DS_PROMETHEUS}",
+    calc="mean",
+    format=UNITS.PERCENT_FORMAT,
+    min=0,
+    max=100,
+    noValue="-",
+    targets=[Target(
+        expr="\n".join([
+            'sum(rate(s3_utilization_service_state{namespace="${namespace}", state="healthy"}[$__rate_interval])) * 100',  # noqa: E501
+            "   /",
+            'sum(rate(s3_utilization_service_state{namespace="${namespace}", state=~"^(unhealthy|healthy)$"}[$__rate_interval]))',  # noqa: E501
+        ]),
+        legendFormat="Online rate",
+    )],
+    thresholds=[
+        Threshold("#808080", 0, 0.0),
+        Threshold("red",     1, 0.0),
+        Threshold("orange",  2, 90.0),
+        Threshold("green",   3, 95.0),
+    ],
+)
+
+
+quotaStatusCode = TimeSeries(
+    title="Quota evaluation status code over time",
+    dataSource="${DS_PROMETHEUS}",
+    fillOpacity=30,
+    lineInterpolation="smooth",
+    unit=UNITS.OPS_PER_SEC,
+    targets=[Target(
+        expr='sum by (code) (rate(s3_quota_evaluation_duration_seconds_count{namespace="${namespace}", code=~"2.."}[$__rate_interval]))',  # noqa: E501
+        legendFormat="Success",
+    ), Target(
+        expr='sum by (code) (rate(s3_quota_evaluation_duration_seconds_count{namespace="${namespace}", code=~"4.."}[$__rate_interval]))',  # noqa: E501
+        legendFormat="Quota Exceeded",
```

Comment:
There is no specific code for quota exceeded, all the errors are quota exceeded ?

What if you receives 400 or 403 or 404 ?

And you don't want to monitor 500 ?

Reply:
QuotaExceeded is 429

We can create two other filters: for the 4.. (not 429) as bad requests, and 5.. as internal service errors. What do you think?

Reply:
Yes I think it could be nice to see different errors

Reply:
Done here: https://github.com/scality/cloudserver/pull/5556/commits/20d2f8bacce5ca1e4b18bbee9747fd8b6a8e88f0#diff-9534a9dab1150e97d07f2e6a3541659b7d37c0aa2fb7db06e95f25fc3864078fR593
We have the success, quota exceeded, bad requests & service errors

Reply:
I wonder if it's better to call them 4xx and 5xx errors, to not mistake a potential 403 for a 400 bad request